org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/DOMElement.java

```diff
@@ -10,6 +10,9 @@
  */
 package org.eclipse.lsp4xml.dom;
 
+import static org.eclipse.lsp4xml.dom.DOMAttr.XMLNS_ATTR;

```

Comment:
Keep the XMLNS_ATTR constants as private and provide a static method DOMAttr#isXmlns(String attributeName) which can be used in DOMAttr#isXmlNs and in DOMElement

Reply:
It is hard to make it private, DOMElement needs this value: https://github.com/angelozerr/lsp4xml/blob/master/org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/DOMElement.java#L123

But I did changes and made static isXmlns(String) and other methods too.